backend/src/main/java/com/festago/student/application/StudentService.java

```diff
@@ -98,4 +100,11 @@ public void verificate(Long memberId, StudentVerificateRequest request) {
         studentRepository.save(new Student(member, studentCode.getSchool(), studentCode.getUsername()));
         studentCodeRepository.deleteByMember(member);
     }
+
+    public StudentResponse findVerification(Long memberId) {
+        Optional<Student> student = studentRepository.findByMemberIdWithFetch(memberId);
+        return student
+            .map(value -> StudentResponse.verified(value.getSchool()))
+            .orElseGet(StudentResponse::notVerified);
+    }
```

Comment:
다음과 같이 라인 수를 줄일 수 있을 것 같네요!
물론 선택입니다. 👍
```suggestion
    public StudentResponse findVerification(Long memberId) {
        return studentRepository.findByMemberIdWithFetch(memberId)
            .map(value -> StudentResponse.verified(value.getSchool()))
            .orElseGet(StudentResponse::notVerified);
    }
```

Reply:
또한 지금 `StudentService` 클래스의 필드 수를 보고 있는데 의존하고 있는 객체가 어마무시하네요.
이 부분에 대해서는 추후 서비스를 작은 서비스로 분리시켜도 좋을 것 같아요!